src/jit/MemoryInl.h

```diff
@@ -378,8 +378,18 @@ static void emitLoad(sljit_compiler* compiler, Instruction* instr)
         JITArg valueArg;
         floatOperandToArg(compiler, operands + 1, valueArg, SLJIT_FR0);
 
-        // TODO: sljit_emit_fmem for unaligned access
-        sljit_emit_fop1(compiler, opcode, valueArg.arg, valueArg.argw, addr.memArg.arg, addr.memArg.argw);
+
+        if (addr.memArg.argw != 0) {
```

Comment:
I think argw is not enough to check this, you would need to compute `base + offset`. It is only enough for immediate offsets

Reply:
The goal was only the immediate case.